### ISSUE TITLE
Game startup error

### ISSUE THREAD

**Issue by SNMetamorph** [LAGonauta/metahook#6]
This error happens on last engine build from Steam (#8196 at this time)
I tested it in Half-Life and in other mods, but error happens in any case.
But in the same time, past versions works normally.
```
Unable to load. Reason:
Failed to create AuxiliaryEffectSlot: Out of memory
```

**Comment by LAGonauta** [comment]
MetaAudio is probably trying to load the basic software OpenAL driver, and I am still to release the version with support for that basic driver. Just put the OpenAL32.dll (OpenAL Soft 32-bit) that I distribute with the release in the same folder where MetaHook.exe is located.

**Comment by LAGonauta** [comment]
@SNMetamorph Should be fixed on v0.0.8, please test.
You can also write "al_version" in the console to see which driver is being used.
